scripts/cleanup.js

```diff
@@ -31,21 +30,15 @@ async function deleteStaleInstances(labelFilter) {
     return labelFilter(instance.metadata.labels);
   });
 
-  return new Promise((resolve, reject) => {
-    async.eachLimit(
-      filtered,
-      5,
-      (instance, callback) => {
-        setTimeout(() => {
-          instance.delete(callback);
-        }, 500); // Delay allows the instance and its databases to fully clear.
-      },
-      (err, res) => {
-        if (err) return reject(err);
-        resolve(res);
-      }
-    );
+  const limit = pLimit(5);
+  const input = [];
```

Comment:
I think this whole thing can be reduced to: 

```js
await Promise.all(
  filtered.map(instance => limit(instance.delete()))
);
```

Reply:
Done!